src/open_inwoner/pdc/migrations/0045_auto_20230220_1628.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2023-02-20 15:28
+
+from django.db import migrations
+
+
+def truncate_product_summary(apps, schema_editor):
+    MyModel = apps.get_model("pdc", "Product")
+    for product in MyModel.objects.all():
+        product.summary = product.summary[:300]
+        product.save()
+
```

Comment:
This save()'s every Product in the database, even if the summary was not too long.